src/nrepl/repl-start.ts

```diff
@@ -69,7 +69,8 @@ async function fetchConfig(configName: string): Promise<DramConfig> {
 
 async function downloadDram(storageUri: vscode.Uri, configPath: string, filePath: string) {
   const downloadUrl = `${dramsBaseUrl()}/${configPath}/${filePath}`;
-  const dirUri = vscode.Uri.joinPath(storageUri, path.join(...path.dirname(filePath).split(/\//)));
```

Comment:
We have had a lot of problems with this category of mistakes. Most oft them we have found, but this one was still lurking in there! (I think we may still have some places undiscovered.)